zerver/models.py

```diff
@@ -2649,7 +2649,10 @@ class UserTopic(models.Model):
     visibility_policy = models.SmallIntegerField(choices=visibility_policy_choices, default=MUTED)
 
     class Meta:
-        unique_together = ("user_profile", "stream", "topic_name")
+        # We have a custom-created UNIQUE index on this table in the migration
+        # 0426_case_insensitive_user_topic_index.py, which takes into account
+        # the case-insensitivity of the topic name.
+        # Thus, the `unique_together` property is not specified here.
```

Comment:
I think you can use the Django constraint system to do this without a custom migration: https://docs.djangoproject.com/en/4.1/ref/models/constraints/

Reply:
(We moved most of our custom indexes to use this a while back).